src/views/bouquets/BouquetDetailView.vue

```diff
@@ -47,6 +47,7 @@ onMounted(() => {
           :description="d.description"
           :img="d.organization.logo"
           :is-markdown="true"
+          :notice="bouquet.extras[`ecospheres:${d.id}:description`]"
```

Comment:
Might be worth having a helper to generate extra key to avoid inconsistencies.

BTW, I could see some topics still have the nested extras you played with before settling on flat keys. The "bouquetti" topic for instance:
```
{
	"extras": {
		"ecospheres": [
			{
				"datasetId": "5a5f4f6c88ee387da4d252a3",
				"descriptions": "with desc"
			}
		],
		"ecospheres:5a5f4f6c88ee387da4d252a3:description": "parceque"
	}
}
```

Reply:
Agreed but refactored in #56 so I suggest we tackle this later on, merging would not be fun

Reply:
And yeah, we'll need to cleanup some topics :p